runtime/gc_vlhgc/SweepHeapSectioningVLHGC.cpp

```diff
@@ -153,6 +156,8 @@ MM_SweepHeapSectioningVLHGC::reassignChunks(MM_EnvironmentBase *env)
 				chunk->chunkBase = (void *)heapChunkBase;
 				chunk->chunkTop = (void *)heapChunkTop;
 				chunk->memoryPool = pool;
+				Assert_MM_true(NULL != pool);
+				chunk->_minFreeSize = OMR_MAX(pool->getMinimumFreeEntrySize(), pool->getSweepPoolManager()->getMinimumFreeSize());
```

Comment:
seems there is some common things in reassignChunks between MM_SweepHeapSectioningVLHGC and MM_SweepHeapSectioningSegmented. Let's try to share it (in base MM_SweepHeapSectioning). Not this PR, but new ones.